pkg/cableengine/healthchecker/statistics_test.go

```diff
@@ -0,0 +1,45 @@
+package healthchecker
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("", func() {
+	Context("Test Statistics", testStatistics)
+})
+
+func testStatistics() {
+	size := 10
+	statistics := &Statistics{
+		size:         uint64(size),
+		previousRtts: make([]uint64, size),
+	}
+
+	sampleSpace := [10]uint64{404351, 490406, 530333, 609556, 609650, 685106, 726265, 785707, 1044609, 1048263}
```

Comment:
some nits sorry ':D
to make the test easier to follow / check on reading

I'd use some constants

```
sampleSpace := [10]uint64{testMinRTT, 490406, 530333, 609556, 609650, 685106, 726265, 785707, testMaxRTT, testLastRTT}

```

(you need to change the max, because it's also the last.)

Reply:
Done.